Source/santasyncservice/syncv1.proto

```diff
@@ -54,7 +54,11 @@ message PreflightRequest {
 }
 
 message PreflightResponse {
```

Comment:
Long as you're in here. We should add an optional machine_id field for GRPC.

Reply:
As discussed, let's do this in a separate PR